doc/validation.md

```diff
@@ -0,0 +1,41 @@
+# Profile Validation
+
+Since version 4.3.14 AutoYaST validates all XML files (profiles,
+rules, classes) before using them to avoid possible problems later.
+
+## Validation Errors
+
+If a XML document does not validate or is not well-formed (contains syntax
+errors) then AutoYaST displays an error popup with details.
+
+It is still possible to continue and use the XML file by clicking the
+`Continue` button. But that is on your risk, the installation might later
+fail or crash, the result will be different than expected or some data
+might be lost.
+
+If you are sure the XML file is correct and the problem is in the validation
+itself you can disable the error popup, see below.
+
+## Manual Validation
+
+Of course, it is much easier to validate the XML files *before* using them.
+The error popup contains some example commands for validating the failed file
+manually. See the [AutoYaST documentation](
+https://doc.opensuse.org/projects/autoyast/#CreateProfile-Manual) for more
+details.
+
+It is recommended to use `jing` for manual validation, it usually produces
+better error messages than `xmllint` (which is also internally used by AutoYaST).
+
+## Disabling Validation
+
+In some rare cases the XML files might work correctly but the validation
+reports errors because of an issue in the schema definition or in the validation
+process itself.
+
+In that case you can set the environment variable `YAST_SKIP_XML_VALIDATION=1`
```

Comment:
np: YAST_SKIP_PROFILE_VALIDATION ? (it is more explict about which 'XML' we are going to validate). Just an idea...

Reply:
Initially I wanted to use something like that, but later I realized that we can also validate the rules and classes. So it's not only about the main XML profile.

But if we consider the rules and classes as a part of the profile (although in different files) I'm fine with that...